consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/validation/RoundChangeCertificateValidator.java

```diff
@@ -40,19 +42,22 @@
   private final Collection<Address> validators;
   private final MessageValidatorForHeightFactory messageValidatorFactory;
   private final BftExtraDataCodec bftExtraDataCodec;
+  private final ProtocolContext protocolContext;
   private final long quorum;
   private final long chainHeight;
 
   public RoundChangeCertificateValidator(
       final Collection<Address> validators,
       final MessageValidatorForHeightFactory messageValidatorFactory,
       final long chainHeight,
-      final BftExtraDataCodec bftExtraDataCodec) {
+      final BftExtraDataCodec bftExtraDataCodec,
+      final ProtocolContext protocolContext) {
```

Comment:
Rather we just pass in the BftBlockInterface instead of the entire ProtocolContext. This is also what are doing in the RoundChangeMessageValidator for example.

Reply:
Done!